services/graph/pkg/service/v0/spacetemplate/.space/readme.md

```diff
@@ -0,0 +1 @@
+Here you can add a description for this Space.
```

Comment:
Web was putting a translated text, based on the language of the user. Can you translate the text backend side as well? `Accept-language` header should be available on the `POST` request that creates the space.

Reply:
That would be a hard requirement to enable this feature 🙋‍♀️

Reply:
Collecting issues with space templates here: https://github.com/owncloud/ocis/issues/8713